README.md

```diff
@@ -13,6 +13,44 @@ composer require webfox/laravel-backed-enums
 
 ## Usage
 
+### Make Command
+
+Conveniently create a new backed enum using the make command.
+
+#### Command:
+
+```Bash
+php artisan make:enum {name} {type?} 
+```
+
+#### Arguments:
+
+* `{name}`: The name of the enum class to be created (e.g., OrderStatus). The command will automatically append "Enum" to the name (e.g., OrderStatusEnum).
+* `{type?}`: The underlying data type for the enum. Can be either --int --string or if not specified it will be a pure enum.
+* `{--force}`: Overwrite the enum if it already exists.
+  Example Usage:
+
+To create an enum named OrderStatusEnum backed by integers:
+
+``` Bash
+php artisan make:enum OrderStatus --int
+```
+
+To create an enum named OrderStatusEnum backed by strings:
+
+``` Bash
+php artisan make:enum OrderStatus --string
+```
+
+To create a pure enum named OrderStatusEnum:
+
+``` Bash
+php artisan make:enum OrderStatus
+
+```
+
+This will generate an OrderStatusEnums in the `app/Enums` directory.
+
 ### Setup your enum
```

Comment:
Maybe this should be something like `Upgrade your existing enums`.
Right now it reads like you need to do this after creating the enum with the command.